www/webapp/src/views/CrudList.vue

```diff
@@ -61,6 +61,7 @@
                         fab
                         depressed
                         :disabled="user.working"
+                        aria-label="Add new record"
```

Comment:
```suggestion
                        aria-label="Add new entry"
```
`CrudList.vue` is generic an e.g. also used in the token management GUI.